### PR TITLE
Added e2e test for operator resolution

### PR DESCRIPTION
**Pull request by dtfranz**
Resolver e2e test for Operator Controller. Creates an Operator Resource specifying prometheus as the packageName then expects resolution to succeed and provide a bundle path.

### CODE REVIEW

File: test/e2e/install_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package e2e
```
Comment (tmshort) [inline test/e2e/install_test.go:1]:
```
package e2e_test
```
??

File: test/e2e/install_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+			err      error
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+			operator = &operatorv1alpha1.Operator{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: fmt.Sprintf("operator-%s", rand.String(8)),
+				},
+				Spec: operatorv1alpha1.OperatorSpec{
+					PackageName: "prometheus",
```
Comment (awgreene) [inline test/e2e/install_test.go:36]: Nit, use a constant.

File: test/e2e/install_test.go (hunk 1 above)
Comment (perdasilva) [reply test/e2e/install_test.go:1]: I think package e2e is fine - there's only tests and test tooling here...no?

File: test/e2e/install_test.go (hunk 2 above)
Comment (dtfranz) [reply test/e2e/install_test.go]: Updated :smiley:

File: test/e2e/install_test.go (hunk 1 above)
Comment (dtfranz) [reply test/e2e/install_test.go:1]: Yeah that's right, I feel like I usually just see that convention when we have `actualpkg` and then when we test it `actualpkg_test`

**Review by grokspawn — APPROVED** [review]

**Pull request merged**